docs/DSP/GameCube_DSP_Users_Manual/GameCube_DSP_Users_Manual.tex

```diff
@@ -198,6 +198,23 @@
   \end{description}
 }
 
+% Command to show flags.
+\newcommand{\DSPOpcodeFlags}[8]
+{
+  \textbf{Flags:}
+  \begin{description}
+  \item \begin{tabular}{|c|c|c|c|c|c|c|c|}
+    \hline
+    OS & LZ & TB & S32 & S  & AZ & O  & C \\
+    \hline
+    #1 & #2 & #3 & #4  & #5 & #6 & #7 & #8 \\
+    \hline
+  \end{tabular}
+  \end{description}
+}
+
+\newcommand{\DSPOpcodeFlagsUnchanged}[0]{\DSPOpcodeFlags{-}{-}{-}{-}{-}{-}{-}{-}}
```

Comment:
I think we should put an explaination somewhere (perhaps in the Opcode syntax chapter, or maybe next to Conditional opcodes?) on what the used symbols (which, as far as I could see, were: "-", "X" and "0").

I'm _assuming_ "X" is "affected by the operation and set accordingly", "0" is "cleared/set to zero" and "-" is "unaffected"; but thats even knowing the DSP.

Reply:
I added a section on flags that documents the way carry and overflow work, and put that information there as well.